Client/Writes/PointData.cs

```diff
@@ -215,25 +246,29 @@ public PointData Timestamp(DateTimeOffset timestamp, WritePrecision timeUnit)
         /// <returns></returns>
         public PointData Timestamp(Instant timestamp, WritePrecision timeUnit)
         {
-            Precision = timeUnit;
-
+            BigInteger? time = null;
```

Comment:
Please remove a redundant initializer => `BigInteger? time;`

Reply:
Can you clarify which  redundant did you mean?
is it line 249? it doesn't seem redundant  to me because it is a local variable not _time which is immutable.

Reply:
The assignment of `null` is unnecessary because is not used in any execution path.